AI/tests/run_backfill.py

```
@@ -0,0 +1,85 @@
+
+# 프로젝트 루트 경로 추가
+current_dir = os.path.dirname(os.path.abspath(__file__))
+project_root = os.path.abspath(os.path.join(current_dir, "../.."))
+if project_root not in sys.path:
+    sys.path.append(project_root)
+
+# daily_routine의 파이프라인 함수를 가져옵니다.
+from AI.pipelines.daily_routine import run_daily_pipeline
+
+def run_backfill(start_date: str, end_date: str, tickers: list, enable_xai: bool):
+    print(f"🚀 백필(Backfill) 시뮬레이션 시작!")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)